src/v0/destinations/marketo_bulk_upload/fileUpload.js

```diff
@@ -143,6 +143,7 @@ const getFileData = async (inputEvents, config, headerArr) => {
     requestTime = endTime - startTime;
     stats.histogram('marketo_bulk_upload_create_file_time', requestTime);
     stats.histogram('marketo_bulk_upload_upload_file_size', fileSize);
+    console.log(fileSize);
```

Comment:
this should done using logger with proper message including what is the destination id etc